content/shmem_quiet.tex

```diff
@@ -19,9 +19,9 @@
 
 \apidescription{
     The \FUNC{shmem\_quiet} routine ensures completion of \PUT{}, \ac{AMO},
-    memory store, and nonblocking \PUT{} and \GET{} routines on
+    \emph{put-with-signal}, memory store, and nonblocking \PUT{},\emph{put-with-signal}, and \GET{} routines on
```

Comment:
The sh_q routine ensure completion of blocking put, amo, pws, and memory store, as well as nonblocking put, pws, and get routines on .....

and -> before memory store
and nonblocking -> as well as nonblocking